### PR TITLE
docs(workflows): update filtered_*.md for filter_DiskANN useage

### PR DESCRIPTION
**Pull request by Coien-rr**
- [ ] Does this PR have a descriptive title that could go in our release notes?
- [ ] Does this PR add any new dependencies?
- [ ] Does this PR modify any existing APIs?
   - [ ] Is the change to the API backwards compatible?
- [ ] Should this result in any changes to our documentation, either updating existing docs or adding new ones?
 
#### What does this implement/fix? Briefly explain your changes.
When I followed the docs to experiment with the siftsmall dataset, i found some mismatch information in docs
1. In filtered_in_memory.md
``` bash
build/apps/utils/compute_groundtruth --data_type float --dist_fn l2 --base_file siftsmall/siftsmall_base.bin --query_file siftsmall/siftsmall_query.bin --gt_file siftsmall/siftsmall_gt_35.bin --K 100 --label_file ./rand_labels_50_10K.txt --filter_label 35 --universal_label 0
```
this util is not able to generate gt file for filtered-DiskANN, so i update it as follow:
``` bash
build/apps/utils/compute_groundtruth_for_filters --data_type float --dist_fn l2 --base_file siftsmall/siftsmall_base.bin --query_file siftsmall/siftsmall_query.bin --gt_file siftsmall/siftsmall_gt_35.bin --K 100 --label_file ./rand_labels_50_10K.txt --filter_label 35 --universal_label 0
```
In addition, I discovered that the search statement was using an index file that didn't match, because build siftsmall_R32_L50_filtered_index for filtered vamana, but use siftsmall_R20_L40_SR32_filtered_index for searching
```bash
build/apps/build_memory_index --data_type float --dist_fn l2 --data_path siftsmall/siftsmall_base.bin --index_path_prefix siftsmall/siftsmall_R32_L50_filtered_index -R 32 --FilteredLbuild 50 --alpha 1.2 --label_file ./rand_labels_50_10K.txt --universal_label 0
build/apps/search_memory_index --data_type float --dist_fn l2 --index_path_prefix data/sift/siftsmall_R20_L40_SR32_filtered_index --query_file siftsmall/siftsmall_query.bin --gt_file siftsmall/siftsmall_gt_35.bin --filter_label 35 -K 10 -L 10 20 30 40 50 100 --result_path siftsmall/filtered_search_results
```
For the purpose of simplifying operations, i save the index file into `siftsmall`, the updated doc as follow:
``` bash
build/apps/build_memory_index  --data_type float --dist_fn l2 --data_path siftsmall/siftsmall_base.bin --index_path_prefix siftsmall/siftsmall_R32_L50_filtered_index -R 32 --FilteredLbuild 50 --alpha 1.2 --label_file ./rand_labels_50_10K.txt --universal_label 0
build/apps/search_memory_index  --data_type float --dist_fn l2 --index_path_prefix siftsmall/siftsmall_R32_L50_filtered_index --query_file siftsmall/siftsmall_query.bin --gt_file siftsmall/siftsmall_gt_35.bin --filter_label 35 -K 10 -L 10 20 30 40 50 100 --result_path siftsmall/filtered_search_results
```
2. In filtered_ssd_index.md 
Similar to the above question, I made some small changes to correspond

### CODE REVIEW

**Comment by Coien-rr** [comment]
> @Coien-rr please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Comment by yashpatel007** [comment]
Let me take a quick look and verify. Thank you.

File: workflows/filtered_in_memory.md [hunk 1]
```diff
@@ -88,18 +89,18 @@ This will genearate label file with 10000 data points with 50 distinct labels, r
...
 ```
 	
 Note that neither approach is designed for use with random synthetic labels, which will lead to unpredictable accuracy at search time.
 
 Now build and search the index and measure the recall using ground truth computed using bruteforce. We search for results with the filter 35.
 ```bash
-build/apps/utils/compute_groundtruth --data_type float --dist_fn l2 --base_file siftsmall/siftsmall_base.bin --query_file siftsmall/siftsmall_query.bin --gt_file siftsmall/siftsmall_gt_35.bin --K 100 --label_file ./rand_labels_50_10K.txt --filter_label 35 --universal_label 0
+build/apps/utils/compute_groundtruth_for_filters --data_type float --dist_fn l2 --base_file siftsmall/siftsmall_base.bin --query_file siftsmall/siftsmall_query.bin --gt_file siftsmall/siftsmall_gt_35.bin --K 100 --label_file ./rand_labels_50_10K.txt --filter_label 35 --universal_label 0
 build/apps/build_memory_index  --data_type float --dist_fn l2 --data_path siftsmall/siftsmall_base.bin --index_path_prefix siftsmall/siftsmall_R32_L50_filtered_index -R 32 --FilteredLbuild 50 --alpha 1.2 --label_file ./rand_labels_50_10K.txt --universal_label 0
```
Comment (yashpatel007) [inline workflows/filtered_in_memory.md:100]: I think the path for label file is off here, It was generated in ```build/apps/utils``` and executed ```build/apps```  in so the label file path needs to be ```./utils/<label_file_name>``` for accuracy

**Review by yashpatel007 — COMMENTED** [review]
Please correct the path for label file while building

File: workflows/filtered_in_memory.md (hunk 1 above)
Comment (yashpatel007) [reply workflows/filtered_in_memory.md:100]: ```build/apps/build_memory_index  --data_type float --dist_fn l2 --data_path siftsmall/siftsmall_base.bin --index_path_prefix siftsmall/siftsmall_R32_L50_filtered_index -R 32 --FilteredLbuild 50 --alpha 1.2 --label_file ./utils/rand_labels_50_10K.txt --universal_label 0```

**Comment by Coien-rr** [comment]
> Please correct the path for label file while building

sry,i neglected to mention the path. I will modify it as soon

File: workflows/filtered_in_memory.md (hunk 1 above)
Comment (Coien-rr) [reply workflows/filtered_in_memory.md:100]: But I tested it and the label file is indeed generated in the root directory

image

**Review by yashpatel007 — APPROVED** [review]

**Pull request closed without merging**